contrib/javaee.wildfly/src/org/netbeans/modules/javaee/wildfly/resources/xml.xsd

```diff
@@ -1,111 +1,107 @@
 <?xml version='1.0'?>
-<!--
```

Comment:
Please keep the license information - it makes it easier to reason about the code and if I remember correctly, we have multiple copies of the xml.xsd and they all should have the header.